code/socialDistribution/views.py

```diff
@@ -165,10 +165,10 @@ def home(request):
 
     # get all friend posts from authors friends
     # this should probably just get from recieved_posts (TODO)
-    for other in author.followers.all():
-        if author.is_friends_with(other):
-            friend_posts = other.posts.listed().get_friend()
-            posts = posts.union(friend_posts)
+    # for other in author.followers.all():
```

Comment:
Should this be commented still/do we have an issue created to add friend/private shared posts back to the homepage?

Reply:
Ya, this can probably be uncommented. I just wasn't sure what we wanted to do with the home page, which can be handled in another issue. But I'll uncomment this for now.

Reply:
Fixed now. Had to add a little error check since it would crash trying to get the posts of Author. In a future PR, might make more sense to only show Public Local posts at this route